src/crypto/index.ts

```diff
@@ -130,13 +130,14 @@ export interface IRoomKeyRequestBody extends IRoomKey {
 }
 
 export interface IMegolmSessionData {
+    [key: string]: any;
```

Comment:
That sounds fairly dangerous? If I understand correctly we can set any property to any value?

Is it by any chance possible to know the type ahead of time here?

Reply:
Well it can be extended with arbitrary fields which are namespaced, js-sdk might not handle all of them but consumers might.